copier.yml

```
@@ -33,6 +33,7 @@ use_windows_in_ci:
     default: no
 
 
+
```

YAMLlint (1.38.0)

[error] too many blank lines (3 > 2)

(empty-lines)